Classes/Controller/AdministrationController.php

```diff
@@ -377,7 +383,11 @@ public function indexAction(): void
         $dblist->allFields = 1;
         $dblist->displayFields = false;
         $dblist->dontShowClipControlPanels = true;
-        $dblist->counter++;
+        //todo can be remove when 10.4 supported dropped
+        $versionInformation = GeneralUtility::makeInstance(\TYPO3\CMS\Core\Information\Typo3Version::class);
```

Comment:
No need to use GU, just `(new \TYPO3\CMS\Core\Information\Typo3Version())->getMajorVersion()` as this isn't a service.